### PR TITLE
Gherkin for pipeline builder to support local tektonhub instances

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-6236
Story: https://issues.redhat.com/browse/ODC-6511

Acceptance criteria:

- Cluster admin can modify the default Tekton Hub API endpoint to point to their local Tekton Hub 

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

### CODE REVIEW

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature [hunk 1]
```diff
@@ -329,4 +329,14 @@ Feature: Create the pipeline from builder page
...
               And user clicks Add task button under Tasks section
               And user types 'git'
-             Then user will see Task, clusterTask only
\ No newline at end of file
+             Then user will see Task, clusterTask only
+
+
+        @regression @manual @odc-6236
+        Scenario: Pipeline builder to support local Tekton Hub instances : P-02-TC21
+            Given user has setup tektonhub insance
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature:337]:
```suggestion
            Given user has setup tektonhub instance
```

Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature [hunk 2]
```diff
@@ -329,4 +329,14 @@ Feature: Create the pipeline from builder page
...
+            # Refer to document https://docs.google.com/document/d/1HImc2DdtFKMWgk5dTm8Ib-I3wgnVXxdHxzX8Cn0jrZA/edit?usp=sharing for setting up tektonhub insance
+              And user is at Pipeline Builder page
+             When user clicks on Add task button under Tasks section
+              And user search task in quick search bar present in tektonhub catalog
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

**Comment by sanketpathak** [comment]
Pr only affects E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**